src/uwtools/config/formats/nml.py

```diff
@@ -1,7 +1,7 @@
 from collections import OrderedDict
 from typing import Optional, Union
 
-import f90nml
+import f90nml  # type: ignore
```

Comment:
The `f90nml` library does not yet provide type-hint stubs for `mypy`. This suppresses a warning from `mypy` about that issue. In the future, we could choose to write our own type stubs for the parts of `f90nml` that we use, if the maintainer does not provide stubs.

Reply:
We'll probably have to do this at some point to be honest